contracts/src/starknet/rewaave/token_bridge.cairo

```diff
@@ -160,6 +139,35 @@ func initiate_withdraw{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_c
     return ()
 end
 
+@external
+func bridge_rewards{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+    l2_token : felt, l1_recipient : felt, amount : Uint256):
+    let (to_address) = get_l1_token_bridge()
+
+    let (l1_token) = l2_token_to_l1_token.read(l2_token)
+
+    let (token_owner) = get_caller_address()
+
+    let (bridge_address) = get_contract_address()
```

Comment:
`bridge_address` not used in the function

Reply:
ack